docs/MultiClusterObservability-CRD.md

```diff
@@ -23,103 +23,184 @@ observability.open-cluster-management.io/v1beta1
    <td><strong>Req’d</strong>
    </td>
   </tr>
+  <tr>
+   <td>EnableDownsampling
+   </td>
+   <td>bool
+   </td>
+   <td>Enable or disable the downsample.
+<p>
+The default value is <strong>true</strong>.
+<p>
+Note: Disabling downsampling is not recommended as querying long time ranges without non-downsampled data is not efficient and useful.
+   </td>
+   <td>N
+   </td>
+  </tr>
   <tr>
    <td>ImagePullSecret
    </td>
    <td>string
    </td>
    <td>Pull secret of the MultiCluster Observability images
    </td>
-   <td>n
+   <td>N
+   </td>
+  </tr>
+  <tr>
+   <td>NodeSelector
+   </td>
+   <td>map[string]string
+   </td>
+   <td>Spec of NodeSelector
+   </td>
+   <td>N
+   </td>
+  </tr>  
+  <tr>
+   <td>ObservabilityAddonSpec
+   </td>
+   <td>ObservabilityAddOnSpec
+   </td>
+   <td>The observabilityAddonSpec defines the global settings for all managed clusters which have observability add-on enabled.
```

Comment:
suggest to replace "enabled" with "installed". Since observabilityAddonSpec has a parameter "enabled", that maybe lead to confusing